ci-utils/tests.sh

```diff
@@ -61,6 +61,9 @@ done
 while sleep 60; do echo "=====[ $SECONDS seconds still running ]====="; done &
 
 CONTAINER=${DOCKER_SERVICE_NAME}.1.$(docker service ps -f "name=${DOCKER_SERVICE_NAME}.1" ${DOCKER_SERVICE_NAME} -q --no-trunc | head -n1)
+
+# Copy vendor/bundle for cache
+docker cp ${CONTAINER}:/srv/app/vendor/bundle vendor/bundle
```

Comment:
if the copying is just for Travis cache then we don't need this either.

Reply:
I removed it.